src/ImGui/Desktop/Fusee.ImGui.Desktop/Templates/ImGuiFilePicker.cs

```diff
@@ -461,33 +440,27 @@ public virtual unsafe void Draw(ref bool filePickerOpen)
             return;
         }
 
-        private List<string> GetFileSystemEntries(string fullName)
+        private List<FileSystemInfo> GetFileSystemEntries(string fullName)
         {
             try
             {
-                var files = new List<string>();
-                var dirs = new List<string>();
+                var folders = new List<DirectoryInfo>();
+                var files = new List<FileSystemInfo>();
 
-                foreach (var fse in Directory.GetFileSystemEntries(fullName, ""))
+                foreach (var f in Directory.GetFileSystemEntries(fullName, ""))
                 {
-                    if (Directory.Exists(fse))
+                    if (Directory.Exists(f))
```

Comment:
I think this may not be the best way to differentate between folder and file. See accepted answer in[ this post](https://stackoverflow.com/questions/1395205/better-way-to-check-if-a-path-is-a-file-or-a-directory).

Reply:
Good catch. Changed it accordingly